src/main/java/org/radarbase/management/service/UserService.kt

```diff
@@ -412,14 +348,13 @@ class UserService @Autowired constructor(
         return userMapper.userToUserDTO(userRepository.findOneWithRolesByLogin(login))
     }
 
-    @Transactional(readOnly = true)
-    /**
-     * Get the current user.
-     * @return the currently authenticated user, or null if no user is currently authenticated
-     */
-    fun getUserWithAuthorities(): User? {
-        return SecurityUtils.currentUserLogin?.let { userRepository.findOneWithRolesByLogin(it) }
-    }
+    @get:Transactional(readOnly = true)
+    val userWithAuthorities: User?
```

Comment:
If we are renaming this I would make it `currentUserWithAuthorities`

```suggestion
    val currentUserWithAuthorities: User?
```